### ISSUE TITLE
emitOnAllTabs - on by default?

### ISSUE THREAD

**Issue by AlonMeytalVia** [SupremeTechnopriest/react-idle-timer#254]
## Question

Hi! First of all, great package?
I was wondering if upgrading from 4.x to 5.x is ok for us since we used `emitOnAllTabs: true`.
The property was deprecated in 5.x, and I was wondering if it was just "set to true" by default and therefore, unnecassery as configuration?? 🤔 

Thanks! 🙏

**Comment by SupremeTechnopriest** [comment]
Yes emitOnAllTabs is the default behavior now. You are good to go!
